task_12/grid.html

```diff
@@ -0,0 +1,414 @@
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+</head>
+<body>
+<main class="menu">
+    <!-- 1 cell -->
+    <div class="menu-item">
+        <div class="menu-item__img">
+            <img src="./image/p_O.jpg" alt="">
+        </div>
+        
+        <div class="menu-item__desc">
+            <div class="menu-item__title indents">
+                <div>
+                    Бургер
+                </div>
+                <div>
+                    <img src="https://api.iconify.design/ic-sharp-highlight-alt.svg?color=green" alt="">
+                </div>
+            </div>
+            <div class="menu-item__price indents">
+                <div>10 р</div>
+                <div>330 г</div>
+            </div>
+        </div>
+        <div class="menu-item__popup__window">
+            Булка, соус терияки в сочетании с сочным ананасом, сыр чеддер,
+            свежий томат и салат Айсберг, густой майонез, котлела из охлажденной
+            говядины.
+        </div>
+
+            <div>
+                <div class="menu-item__button indents">Order</div>
+            </div>
+    </div>
+
+    <!-- 2 cell -->
+    <div class="menu-item">
+        <div class="menu-item__img">
+            <img src="./image/100608_picture_XNlaEKL.jpg" alt="">
+        </div>
+        <div class="menu-item__desc">
+            <div class="menu-item__title indents">
+                <div>
+                    Чизбургер
+                </div>
+                <div>
+                    <img src="https://api.iconify.design/ic-sharp-highlight-alt.svg?color=green" alt="">
+                </div>
+            </div>
+
+            <div class="menu-item__price indents">
+                <div>4 р</div>
+                <div>100 г</div>
+            </div>
+        </div>
+
+        <div class="menu-item__popup__window">
+            Булка, соус терияки в сочетании с сочным ананасом, сыр чеддер,
+            свежий томат и салат Айсберг, густой майонез, котлела из охлажденной
+            говядины.
+        </div>
+
+        <div>
+            <div class="menu-item__button indents">Order</div>
+        </div>
+    </div>
+    <!-- 2 cell -->
+
+    <!-- 3 cell -->
+    <div class="menu-item menu-item_size_x">
+        <div class="menu-item__img">
+            <img src="./image/a94f5f9a9c0a.jpg" alt="">
+        </div>
+
+        <div class="menu-item__desc">
+            <div  class="menu-item__title indents">
+                <div>
+                    Пица
+                </div>
+                <div>
+                    <img src="https://api.iconify.design/ic-sharp-highlight-alt.svg?color=green" alt="">
+                </div>
+            </div>
+
+            <div class="menu-item__price indents">
+                <div>14 р</div>
+                <div>740 г</div>
+            </div>
+        </div>
+
+        <div class="menu-item__popup__window">
```

Comment:
Incorrect name
_menu-item__popup-window_ is correct

Reply:
correct